generic_tracers/generic_COBALT.F90

```diff
@@ -1521,70 +1501,130 @@ subroutine user_add_params(param_file)
+    ! Nitrification as in Paulot et al., 2020 (https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2019MS002026)
+    ! Note: Values and functional form to be updated for ESM4.5 following the data compilation of Tang et al.,
+    ! (https://essd.copernicus.org/articles/15/5039/2023/essd-15-5039-2023.html)
+    call get_param(param_file, "generic_COBALT", "gamma_nitrif", cobalt%gamma_nitrif, "nitrification rate constant", &
+                   units="(moles kg)-1 sec-1", default= 3.5e6/(30.0*sperd))
+    call get_param(param_file, "generic_COBALT", "knh3_nitrif", cobalt%k_nh3_nitrif, "nitrification half-saturation", &
+                   units="mol kg-1", default=3.1e-9)
+    call get_param(param_file, "generic_COBALT", "irr_inhibit", cobalt%irr_inhibit, &
+                   "half-saturation for light inhibition of nitrification", units="W m-2", default=10.0)
+    call get_param(param_file, "generic_COBALT", "k_o2_nit", cobalt%k_o2_nit, &
+                   "oxygen half-saturation constant for nitrification", units="mol O2 kg-1", default= 3.9e-6)
+    call get_param(param_file, "generic_COBALT", "o2_min_nit", cobalt%o2_min_nit, &
+                   "Minimum oxygen level for nitrification", units="mol O2 kg-1", default=0.01e-6)
+    ! Anammox parameterization developed for ESM4.5.  This relatively new process is turned off in the default CEFI
+    ! configuration by setting the rate constant to 0.  To activate, set this constant to 0.07 day-1.  Translated to
+    ! sec-1 by the model
+    call get_param(param_file, "generic_COBALT", "gamma_nh4amx", cobalt%gamma_nh4amx, "annamox rate constant", &
+                   units="day-1", default=0.0, scale=I_sperd)
+    call get_param(param_file, "generic_COBALT", "o2_max_amx", cobalt%o2_max_amx, &
+                   "maximum o2 concentration for anammox to occur", units="mol O2 kg-1", default=4.0e-6)
     call get_param(param_file, "generic_COBALT", "k_no3_amx", cobalt%k_no3_amx, &
-           "nitrate half-saturation for anammox", units="mol NO3 kg-1", default= 1.0e-6)
-    call get_param(param_file, "generic_COBALT", "k_o2_nit",         cobalt%k_o2_nit,          "k_o2_nit",         units="mol O2 kg-1   ", default= k_o2_nit)                    ! mol O2 kg-1
-    call get_param(param_file, "generic_COBALT", "o2_min_nit",       cobalt%o2_min_nit,        "o2_min_nit",       units="mol O2 kg-1   ", default= o2_min_nit )                 ! mol O2 kg-1
+                   "nitrate half-saturation for anammox", units="mol NO3 kg-1", default= 1.0e-6)
     !
     !-----------------------------------------------------------------------
     ! Miscellaneous
     !-----------------------------------------------------------------------
     !
-    call get_param(param_file, "generic_COBALT", "tracer_debug",  cobalt%tracer_debug, "tracer_debug", default=.false.)
+    ! Unused?
+    call get_param(param_file, "generic_COBALT", "tracer_debug",  cobalt%tracer_debug, &
```

Comment:
It appears this is not currently being used. We can retain it for now and reconsider removing it later if no one finds it essential. However, I believe it could still be valuable for ongoing code development.